From: Simon McVittie <smcv@debian.org>
Date: Wed, 6 May 2015 09:17:06 +0100
Subject: Doxyfile.in: do not put timestamps in HTML, for reproducible builds

---
 Doxyfile.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Doxyfile.in b/Doxyfile.in
index f0a37ed..dbbdb4f 100644
--- a/Doxyfile.in
+++ b/Doxyfile.in
@@ -102,6 +102,7 @@ DISABLE_INDEX          = NO
 ENUM_VALUES_PER_LINE   = 4
 GENERATE_TREEVIEW      = NO
 TREEVIEW_WIDTH         = 250
+HTML_TIMESTAMP         = NO
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
 #---------------------------------------------------------------------------
